Conceptpower+Spring/src/main/webapp/WEB-INF/views/layouts/concepts/openrequests.html

```diff
@@ -0,0 +1,168 @@
+<html
+   xmlns:th="http://www.thymeleaf.org"
+   xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" 
+   layout:decorate="~{layouts/main}"
+   >
+   <head>
+      <link th:href="@{/resources/assets/css/reviewrequest.css}" rel="stylesheet" />
+      <script>
+      	$(document).on('click', '#conceptDetails', function(e) {
+    	    var conceptid = $(this).data('conceptid');
+
+    	    $.ajax({
+    	        type: "GET",
+    	        url: "[[@{'/conceptDetail'}]]",
```

Comment:
I believe you dont' need the `"`; Thymeleaf should take care of that when you use `[[...]]` (instead of `[(...)]`

Reply:
I think it's true if we use it inside the HTML. Using it in jQuery without the double quotes raises an error.

Reply:
Are you sure? Have you tried it? This says otherwise:
https://www.thymeleaf.org/doc/tutorials/3.0/usingthymeleaf.html (12.3 JavaScript inlining)

Would be good to know if that is wrong.

Reply:
Actually, never mind. Apparently for links things work a little different (https://github.com/thymeleaf/thymeleaf/issues/84)

Reply:
Because when I try without the double quote, I'm clearly getting an error in the console.

Uncaught SyntaxError: Invalid regular expression flags